bindings-go/apis/v2/componentdescriptor.go

```diff
@@ -377,6 +384,9 @@ type ComponentReference struct {
 	// ExtraIdentity is the identity of an object.
 	// An additional label with key "name" ist not allowed
 	ExtraIdentity Identity `json:"extraIdentity,omitempty"`
+	// Digest is the optional digest of the referenced component.
+	// +optional
+	Digest DigestSpec `json:"digest,omitempty"`
```

Comment:
maybe we should add here that this is always set by us in contrast to the resource where it can be set by the user.